mapswipe_workers/mapswipe_workers/utils/api_calls.py

```diff
@@ -13,7 +27,70 @@ def geojsonToFeatureCollection(geojson):
     return geojson
 
 
-def ohsome(request: dict, area: str, properties=None):
+def query_osm(changeset_id: int):
+    """Get data from changesetId."""
+    url = OSM_API_LINK + f"changeset/{changeset_id}.json"
+
+    response = retry_get(url)
+
+    if response.status_code != 200:
+        err = f"osm request failed: {response.status_code}"
+        logger.warning(f"{err}")
+        logger.warning(response.json())
+        raise CustomError(err)
+    response = response.json()["elements"][0]
+    return response
+
+
+def add_to_properties(attribute: str, feature: dict, new_properties: dict):
+    """Adds attribute to new geojson properties if it is needed."""
+    if attribute != "comment":
+        new_properties[attribute.replace("@", "")] = feature["properties"][attribute]
+    else:
+        new_properties[attribute.replace("@", "")] = feature["properties"]["tags"][
+            attribute
+        ]
+    return new_properties
+
+
+def remove_noise_and_add_user_info(json: dict) -> dict:
+    """Delete unwanted information from properties."""
+    logger.info("starting filtering and adding extra info")
+
+    wanted_attributes = ["@changesetId", "@lastEdit", "@osmId", "@version", "source"]
```

Comment:
any reason that here you don't have `comment`, but you have it in the missing_rows dictionary below?